doc-content/shared-kie-docs/src/main/asciidoc/KieServer/controller-java-api-examples-ref.adoc

```diff
@@ -158,25 +122,26 @@ public class RESTTimeoutExample {
                                                                                           MarshallingFormat.JSON,
                                                                                           configuration);
```

Comment:
This command is not formatted well in the resulting doc - it looks like there is an extra line after each parameter.

Reply:
This should be resolved in the actual published output in the customer portal. Code snippets look pretty poor in the doc previews.